### ISSUE TITLE
SWIG - Send image to stdout

### ISSUE THREAD

**Issue by richie21** [mapserver/mapserver#5840]
Hi. This request is to fill a gap between PHPMapScript and SWIG.

With PHP MapScript, the [saveImage([string filename, MapObj oMap])](https://mapserver.org/fr/mapscript/php/phpmapscript.html#id15) method with no argument exists and allows to send image to stdout.

```
 int saveImage([string filename, MapObj oMap])

    Writes image object to specified filename. Passing no filename or an empty filename sends output to stdout
```

In MapServer 7.4, the equivalent with SWIG seems to be [write( [ FILE file=NULL ] )](https://mapserver.org/fr/mapscript/mapscript.html#imageobj-methods). However this method is only supported by Python and C# according to the documentation.

Workaround :

```php
$tmpFilename = tempnam($mapObj->web->imagepath, 'tmp');

// save the image to a temporary file on the hard disk
$imageObj->save($tmpFilename);

// read temporary file and writes it to the output buffer. 
readfile($tmpFilename);

// delete temporary file
unlink($tmpFilename);
```

**Comment by geographika** [comment]
@richie21 - the Python `write` function extends the `imageObj` in MapScript. You can see the code [here](https://github.com/mapserver/mapserver/blob/master/mapscript/python/pyextend.i#L324). 

A similar extend function would need to be added to `mapscript/phpng/php7module.i`. 

You could maybe base it on the PHPMapScript [write function here](https://github.com/mapserver/mapserver/blob/master/mapscript/php/image.c#L258). Although I have no idea about PHP and C integration. You may want to look at the [SWIG PHP docs](http://www.swig.org/Doc4.0/Php.html).

**Comment by jmckenna** [comment]
This has been just fixed by @bragef (#5943) and merged into master.  Please re-try your tests with today's git-master; I confirmed the fix on Windows with PHP  7.2.

**Comment by richie21** [comment]
Thanks @bragef. `echo $image->getBytes();` works. That writes the image data directly to stdout.
